tooling/karaf-maven-plugin/src/main/java/org/apache/karaf/tooling/RunMojo.java

```diff
@@ -141,6 +145,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
             Object featureService = findFeatureService(bundleContext);
             addFeatureRepositories(featureService);
+            fixProjectArtifact();
```

Comment:
I would avoid to introduce such method name here. Why not simply adding a comment with the code inline ?